ads-cert-examples/java/src/main/java/Main.java

```diff
@@ -0,0 +1,108 @@
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.Security;
+import java.security.Signature;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.bouncycastle.asn1.ASN1ObjectIdentifier;
+import org.bouncycastle.asn1.x9.ECNamedCurveTable;
+import org.bouncycastle.asn1.x9.X9ECParameters;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.openssl.PEMKeyPair;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter;
+import org.bouncycastle.util.io.pem.PemObject;
+
+import org.apache.commons.codec.binary.Base64;
+
+public class Main {
+    private static PrivateKey privateKey;
+    private static PublicKey publicKey;
+
+    static {
+        Security.addProvider(new BouncyCastleProvider());
+    }
+
+    public static void main(String[] args) {
+        try {
+            loadKeys(args);
+            String message = String.join(":",
+                    Stream.of(
```

Comment:
`Stream.of` was copied from my previous implementation, kept it assuming the user will be putting in objects instead of raw strings.